packages/terra-data-grid/src/DataGrid.jsx

```diff
@@ -65,6 +65,11 @@ const propTypes = {
    */
   columnHeaderHeight: PropTypes.string,
 
+  /**
+   * Numeric increment in pixels to adjust column width when resizing via the keyboard
+   */
+  columnResizeIncrement: PropTypes.number,
```

Comment:
With the introduction of this prop, is the `onColumnResize` prop still needed? I could not find it in any test or example.

Reply:
@cm9361 thoughts?

Reply:
Consumers new the onColumnResize so that they can respond to the column being resized.  They might want to store preferences or make modifications to the new size or update the size of other columns, for example.  Yes, we need the onColumnResize prop.